packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -286,6 +292,7 @@ export default class TableResize implements EditorPlugin {
             this.currentInsertTd = td;
             if (this.currentInsertTd) {
                 this.inserter = this.createInserter(tableRect);
+                this.inserter.addEventListener('mouseout', this.onMouseOutInserter);
```

Comment:
let's move this inside createInserter where we add the click event handler as well.